src/main/java/uk/gov/digital/ho/pttg/api/IpsStatisticsResource.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.digital.ho.pttg.api;
+
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import uk.gov.digital.ho.pttg.IpsStatisticsService;
+
+import java.time.LocalDate;
+
+import static net.logstash.logback.argument.StructuredArguments.value;
+import static uk.gov.digital.ho.pttg.IpsStatisticsService.NO_STATISTICS;
+import static uk.gov.digital.ho.pttg.application.LogEvent.*;
+
+@AllArgsConstructor
+@Slf4j
+public class IpsStatisticsResource {
```

Comment:
Comment out the @RestController with a TODO?  It lets the next developer know that this was omitted on purpose.

Reply:
Sounds like a plan